app/src/component/transactionList/internal/transaction-list-item.js

```diff
@@ -27,7 +34,12 @@ class TransactionListItem extends React.PureComponent {
         </View>
         <View style={transactionListStyle.row}>
           <View style={transactionListStyle.col}>
-            <Text style={[transactionListStyle.smallText, transactionListStyle.txid]}>{txid.substring(0, 8)}</Text>
+            <Link text={txid.substring(0, 8)} style={transactionListStyle.smallLink} onPress={() =>
+              Linking.openURL(`https://explorer.lbry.io/tx/${txid}`).catch(err => notify({
```

Comment:
I think it would make more sense to add to the `<Link />` component. Just use an `href` prop or something. You could also move the `notify` call and just use an `errorMessage` prop.